src/kern/arm/cpu-arm.cpp

```diff
@@ -340,6 +340,18 @@ EXTENSION class Cpu
   };
 };
 
+// ------------------------------------------------------------------------
+INTERFACE [arm && cpu_virt && arm_cortex_r52]:
```

Comment:
If `arm && cpu_virt && mpu` is true, the `Imp_csctlr` constant must be defined because it is referenced from `init_sctlr` (even if unused). Right now, the R52 is the only 32 bit CPU with MPU, so that doesn't make a difference ATM. But as we add other processors in the future, it's better to already add the fallback:

```c++
// ------------------------------------------------------------------------
INTERFACE [arm && cpu_virt && !arm_cortex_r52]:

EXTENSION class Cpu
{
  enum { Imp_csctlr = 0 };
};
```

Likewise, the constant does not need to be public in the other case. It's only used by the `Cpu` class internally.

Reply:
yikes, I overlooked that. Thanks for catching it! I've corrected it now.